extensions/amp-gwd-animation/0.1/amp-gwd-animation-impl.js

```diff
@@ -498,5 +498,6 @@ export class AmpGwdRuntimeService {
  * @param {!Element} element
  */
 function reflow(element) {
-  element./*OK*/offsetWidth = element./*OK*/offsetWidth;
+  Object.defineProperty(
```

Comment:
Spoke with @nainar about this, `Element.offsetWidth` is only a getter – this actually reflowed because it asked for `offsetWidth`.

Since accessing `offsetWidth` should be enough to cause reflow, can we just return the value instead of using `Object.defineProperty`?

```
function reflow(element) { 
  return element.offsetWidth;
}
```